messaging/src/main/java/org/axonframework/eventhandling/pooled/SplitTask.java

```diff
@@ -121,6 +133,10 @@ private boolean splitAndRelease(Segment segmentToSplit) {
             logger.info("Processor [{}] successfully split {} into {} and {}.",
                         name, segmentToSplit, splitStatuses[0].getSegment(), splitStatuses[1].getSegment());
         });
+
+        // Remove the segment from the releases deadlines to allow the Coordinator to claim the split segments
```

Comment:
The code only removes from releasesDeadlines on successful completion. 
If the task fails or throws an exception, segments remain blocked for 60 seconds. Maybe we should put it in catch or finally block?